src/BuildIt.ML/Samples/BuildIt.ML.Sample.UI/BuildIt.ML.Sample.UI.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Forms" Version="3.1.0.637273" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\BuildIt.ML.Sample.Core\BuildIt.ML.Sample.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="App.xaml">
```

Comment:
Are these needed? I thought that these includes aren't required any more

Reply:
no they're not, the entries got added when i had to mess with adding pages in since VS wouldn't create the projects for me in a nice way